lib/dialects/mssql/data-types.js

```diff
@@ -153,6 +153,10 @@ var FLOAT = function() {
 };
 util.inherits(FLOAT, BaseTypes.FLOAT);
 
+BaseTypes.ENUM.prototype.toSql = function() {
+  return 'VARCHAR(10)';
```

Comment:
While you're at it, could you make this `VARCHAR(255)` :) https://github.com/sequelize/sequelize/issues/3967